system/lib/libcxxabi/src/cxa_exception.cpp

```diff
@@ -644,6 +644,14 @@ void __cxa_rethrow() {
     }
 #ifdef __USING_SJLJ_EXCEPTIONS__
     _Unwind_SjLj_RaiseException(&exception_header->unwindHeader);
+#elif __USING_WASM_EXCEPTIONS__
```

Comment:
How about making this `#elif defined(__USING_WASM_EXCEPTIONS__) && !defined(NDEBUG)` to avoid inner if/else/endif?

Reply:
How? We have
```cpp 
#else
    _Unwind_RaiseException(&exception_header->unwindHeader);
#endif
```
at the end so I'm not sure how to rewrite this without an inner `if`...

Reply:
I was thinking you could write this to avoid the duplication of the ` _Unwind_RaiseException(&exception_header->unwindHeader);` block?  And instead rely on the fall though to that `else` case?

Reply:
Aha, sorry, I was confused and thought `NDEBUG` as the opposite... Simplified it, and also fixed the existing `__cxa_throw` part, which I copied this code from.